roles/nuage_master/tasks/main.yaml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Install Nuage Kubemon
+  sudo: true
+  yum: name={{ nuage_kubemon_rpm }} state=present
+
+- name: set kubeconfig 
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^kubeconfig line='kubeconfig{{':'}} {{ admin_config }}'
+
+- name: set openshift admin 
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^openshiftAdmin line='openshiftAdmin{{':'}} {{ os_admin }}'
+
+- name: Set Openshift master config path
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^openshiftMasterConfig line='openshiftMasterConfig{{':'}} {{ master_config_yaml }}'
```

Comment:
@abhat correct, we wouldn't want to rely on direct access to etcd.

@abhat: You can use openshift.master.sdn_cluster_network_cidr and openshift.master.sdn_host_subnet_length to get the values from openshift_facts instead of querying the master config.

@rajatchopra Is there anything in OpenShift that uses the clusterNetworkCIDR, hostSubnetLength, and networkPluginName within OpenShift outside of openshift-sdn? Or can those be omitted in the case of third-party networking plugins?